web-app/CKS.Web.Test/IntegrationTests/SearchTests.cs

```diff
@@ -11,19 +18,67 @@ public class SearchTests : TestBase
 		public SearchTests(ITestOutputHelper output) : base(output)
 		{ }
 
-		[Obsolete("Please replace this test with a real test when the search endpoint is setup")]
+
+		private HttpClient SetupClientWithFakeSearchProvider(string fakeSearchResultsFilePath = null)
+		{
+			var searchResults = new SearchResults();
+			if(fakeSearchResultsFilePath != null)
+				searchResults = JsonConvert.DeserializeObject<SearchResults>(File.ReadAllText(fakeSearchResultsFilePath));
+
+			var mockSearchProvider = new Mock<ISearchProvider>();
+			mockSearchProvider.Setup(x => x.Search(It.IsAny<ISearchUrl>())).Returns(searchResults);
+			var client = _factory
+				.WithImplementation<ISearchProvider>(mockSearchProvider.Object)
+				.CreateClient();
+
+			return client;
+		}
+
 		[Fact]
 		public async void ReturnsJson()
 		{
 			// Arrange
-			var client = _factory
-			   .CreateClient();
+			var client = SetupClientWithFakeSearchProvider();
 
 			// Act
 			var response = await client.GetAsync("/api/search");
 
+			// Assert
 			response.StatusCode.ShouldBe(System.Net.HttpStatusCode.OK);
 			response.Content.Headers.ContentType.MediaType.ShouldBe("application/json");
 		}
+
+
+		[Fact]
+		public async void SearchForCancerReturnsCorrectJson()
+		{
+			// Arrange
+			var client = SetupClientWithFakeSearchProvider(@"./IntegrationTests/Fakes/FakeSearchResult-Cancer.json");
+
+			//Act
+			var response = await client.GetAsync("/api/search?q=cancer");
+			var responseBody = await response.Content.ReadAsStringAsync();
+			responseBody = Formatter.FormatJson(responseBody);
+
+			//Assert
+			response.StatusCodeIs(HttpStatusCode.OK);
+			responseBody.ShouldMatchApproved();
+		}
+
+		[Fact]
+		public async void FailSearchRespondsCorrectly()
```

Comment:
If we have the above integration tests AND we have the unit tests that test the failing logic, then I don't think we need this integration test too.